COMMERCE-10761

elocamp opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

katienesterovich commented:
ci:test:sf

katienesterovich commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-2.liferay.com/job/test-portal-source-format/1872/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-2.liferay.com//userContent/jobs/test-portal-source-format/builds/1872/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:sf - 0 out of 1 jobs passed in 2 minutes 19 seconds 198 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/37b13847aa7361cd42d316e78a03304c2c470182">37b13847aa7361cd42d316e78a03304c2c470182</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/katienesterovich/liferay-portal/tree/LPS-103270">LPS-103270</a><br/>Branch GIT ID: <a href="https://github.com/katienesterovich/liferay-portal/commit/7613a127a1541a5ae89162c5260ee51012c9f202">7613a127a1541a5ae89162c5260ee51012c9f202</a></p>0 out of 1jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-2.liferay.com/job/test-portal-source-format/1872/"><strike><strong>test-portal-source-format</strong></strike></a></li></ul></div><h5>For more details click <a href="https://test-1-2.liferay.com/userContent/jobs/test-portal-source-format/builds/1872/jenkins-report.html">here</a>.</h5><pre><code>     [java] java.lang.Exception: Found 2 formatting issues:
     [java] 1: Unprocessed exception 'nfe', see LPS-36174: ./modules/apps/asset/asset-auto-tagger-service/src/main/java/com/liferay/asset/auto/tagger/internal/configuration/persistence/listener/AssetAutoTaggerCompanyConfigurationModelListener.java 81 (Checkstyle:UnprocessedExceptionCheck)
     [java] 2: &gt; 80: ./modules/apps/asset/asset-auto-tagger-service/src/main/java/com/liferay/asset/auto/tagger/internal/configuration/persistence/listener/AssetAutoTaggerSystemConfigurationModelListener.java 65 (SourceCheck:JavaLongLinesCheck)
     [java] 
     [java] 	at com.liferay.source.formatter.SourceFormatter.format(SourceFormatter.java:438)
     [java] 	at com.liferay.source.formatter.SourceFormatter.main(SourceFormatter.java:266)
[stopwatch] [run.batch.test.action: 1:06.852 sec]
     [echo] The following error occurred while executing this line:
     [echo] /opt/dev/projects/github/liferay-portal/portal-impl/build.xml:693: Java returned: 1
      [get] Getting: http://test-1-2/job/test-portal-source-format/1872//consoleText
      [get] To: /opt/dev/projects/github/liferay-portal/20191018143957779.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/20191018143957779.txt
  [typedef] Could not load definitions from resource org/apache/maven/artifact/ant/antlib.xml. It could not be found.
  [taskdef] Could not load definitions from resource org/jacoco/ant/antlib.xml. It could not be found.
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null401621915.properties

merge-test-results:
    [mkdir] Created dir: /opt/dev/projects/github/liferay-portal/test-results
[junitreport] Processing /opt/dev/projects/github/liferay-portal/test-results/TESTS-TestSuites.xml to /tmp/null1296626873
[junitreport] Loading stylesheet jar:file:/opt/java/ant/lib/ant-junit.jar!/org/apache/tools/ant/taskdefs/optional/junit/xsl/junit-frames.xsl
[junitreport] Transform time: 283ms
[junitreport] Deleting: /tmp/null1296626873
     [echo] A report with all the test results can be found at test-results/html/index.html.
    [mkdir] Created dir: /opt/dev/projects/github/liferay-jenkins-ee/test-results
     [copy] Copying 1 file to /opt/dev/projects/github/liferay-jenkins-ee/test-results
     [echo] run.batch.test.tear.down.start.timestamp: 10-18-2019 07:40:01:416 PDT
[stopwatch] [run.batch.test.tear.down: 0.001 sec]
     [echo] 
     [echo] Ant GC log:
     [echo] </code></pre></details></html>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest(master)/7548/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-20.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7548/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 0 out of 1 jobs passed in 1 minute 28 seconds 884 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/22cde9ad7563e8696feb1e27b1c1c6367098ddfb">22cde9ad7563e8696feb1e27b1c1c6367098ddfb</a></p>0 out of 1 jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest(master)/7548/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li></ul></div><h5>For more details click <a href="https://test-1-20.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7548/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest(master)/7548/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><div><p>The sender branch SHA could not be found on <strong><a href="https://github.com/katienesterovich/liferay-portal/tree/LPS-103270">katienesterovich/LPS-103270</a></strong>. The sender branch may have been force pushed or deleted after the pull request test was initiated.</p><pre><code>fatal: Not a valid branch point: '7613a127a1541a5ae89162c5260ee51012c9f202'.</code></pre></div></div></li></ol><h4>For upstream results, click <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">here</a>.</h4></details></html>

katienesterovich commented:
ci:stop

ealonso committed:
LPS-175504 Removes soy dependency from modules, since it is not used

brianikim committed:
COMMERCE-10639 Improve SEO for productFriendlyURL

Commit:
LPS-173984 Convert method reference to ServiceTrackerMap

Stephy6 committed:
LPS-174317 Use ClayDatePicker instead of custom date picker

Commit:
LPS-174124 replace stream with TransformUtil.transformToArray

Commit:
LPS-174124 replace IntStream and LongStream with ArrayUtil.toArray()

Commit:
COMMERCE-10627 appending the namespace to the id to fix the issue with filtering

Stephy6 committed:
LPS-174317 Use OnBlur callback function to save the value also when the input loses focus

Stephy6 committed:
LPS-174317 Check that the input value has changed before saving it so we save up requests and use propertyType to know what type of Date is being handled

ling-alan-huang committed:
POSHI-473 Validate that do not use double quotes to wrap multiline literal string

pat270 committed:
LPS-168540 Screen reader should announce the role title when Choose button is focused

Commit:
LPS-173507 Update TEXT_BUTTON path

Commit:
LPS-173507 Add new macros

Commit:
LPS-173507 Add test PRMMDFRequest#DetailPageDisplaysAllMDFInformation

JerryAdrianoo committed:
LPS-174781 Fix PicklistTests

Commit:
LPS-162129 Remove redundant artifacts from compile only BOM

Stephy6 committed:
LPS-173028 Tests update

Commit:
COMMERCE-10627 fixing the issue with container being null when a widget is added to a new page

nNilton committed:
LPS-173287 rename all methods

nNilton committed:
LPS-173287 rename test-bundle to test-bundle-1

nNilton committed:
LPS-173287 create empty test-bundle-2

nNilton committed:
LPS-173287 import bundles

nNilton committed:
LPS-173287 initialize test bundles and add test to update

allen-nagtalon committed:
LPS-174856 Add path to target any disabled button

allen-nagtalon committed:
LPS-174856 Change test intent to match requested behavior

allen-nagtalon committed:
LPS-174856 Separate macros into proper task blocks

allen-nagtalon committed:
LPS-174856 Remove excess teardown steps

nNilton committed:
LPS-173287 _assertAccounts2

nNilton committed:
LPS-173287 auto SF

CalumR23 committed:
POSHI-476 Add method to grab table

CalumR23 committed:
POSHI-476 Replace BaseTable with DefaultTable

Commit:
LPS-175212 Replace Stream usage with TransformUtil.transform

Commit:
LPS-175212 Add dependencies

Commit:
LPS-175213 Replace Stream usage with for loop

Commit:
LPS-174878 Add javax.annotation-api in release.api jar

Commit:
LPS-174878 Update annotation-api version by executing SF

nNilton committed:
LPS-173287 resolve conflict

nNilton committed:
LPS-173287 let's rename methods on demand

nNilton committed:
LPS-173287 fix leftovers

nNilton committed:
LPS-173287 simplify

Stephy6 committed:
LPS-174317 Use output format for the values that are sent back to the server and input for the ones displayed

Commit:
LPS-173507 SF

nNilton committed:
LPS-173287 auto SF

marcelabc committed:
LPS-174159 Adapt how we handle the Picklist field type conversion into InfoFieldValue by using KeyLocalizedLabelPair to represent the selected option

patriciajeanperez committed:
LRQA-79325 Add missing components including upgrades

julschong committed:
LPS-175350 Remove Stream usages in DDMFormImporter

AliciaGarciaGarcia committed:
LPS-174415 on download the default cache value is not cache

AliciaGarciaGarcia committed:
LPS-174415 the adaptive media should have the same behavior with cache and download

cgoncas committed:
LPS-175384 New FilterByCountIndividualSegmentsExpressionVisitorImpl class that implements a IndividualSegmentsExpressionBaseVisitor for filterByCount expressions.

cgoncas committed:
LPS-175384 Add tests for FilterByCountIndividualSegmentsExpressionVisitorImpl.

Stephy6 committed:
LPS-174317 Use ariaLabels correctly, buildLang and change date-fns library usage-

Brunno0liveira committed:
COMMERCE-10639 JSONProductAPI Macro update

Brunno0liveira committed:
COMMERCE-10639 CommerceEntry Path

Brunno0liveira committed:
COMMERCE-10639 CanViewLanguageTagsOnSiteMapsPage test

shuyangzhou committed:
LPS-175888 Remove WAB colliding processing, it never worked.

Commit:
LPS-161981 batch-engine - technical json - add actions so machins understand where to submit this batch file

Commit:
LPS-173507 SF

nNilton committed:
LPS-173287 wordsmith

shuyangzhou committed:
LPS-175889 Avoid unnecessary jmx cleanup during shutdown

patriciajeanperez committed:
LRQA-79325 Include other databases for usm upgrade test suite

julschong committed:
LPS-172315 Use TransformUtil.toLongArray instead of manual for loop

drewbrokke committed:
LPS-168452 portal-bundle-blacklist-test: makes the test configuration multi-location

PROBLEM:

If a configuration is not given a location when it is created, it will be bound
to the bundle location of the first ManagedService service that is found.

Before the changes of LPS-162830, there was no ManagedService, so the
configuration would not be dynamically bound.

After the changes of LPS-162830, a ManagedService is registered for each
annotated configuration found in a bundle. This causes the configuration with no
location to be bound to the MangagedService's bundle in
com.liferay.portal.configuration.settings. This means that only listeners that
match that bundle will receive configuration updates.

The test waits for an update to the BundleBlacklist service whenever the
properties of the config are updated, but since the config is now bound to the
com.liferay.portal.configuration.settings bundle, the BundleBlacklist service
will no longer receive updates, and the test waits forever.

SOLUTION:

The solution to this is to pass in the "?" location when retrieving the
configuration. This avoids the dynamic binding and allows a listener in any
bundle to receive updates from this configuration.

See https://docs.osgi.org/specification/osgi.cmpn/7.0.0/service.cm.html#d0e11038

julschong committed:
LPS-172617 Use TransformUtil.toLongArray in AssetRendererFactoryRegistryUtil

julschong committed:
LPS-174233 Use TransformUtil.toLongArray instead of manual for-loop

tinatian committed:
LPS-173984 Use api method rather than component property, prepare for next

tinatian committed:
LPS-173984 Inline the logic to the only usage to simplify the logic

tinatian committed:
LPS-173984 Use constants

tinatian committed:
LPS-173984 Baseline

tinatian committed:
LPS-173984 Fix unit test

tinatian committed:
LPS-173984 No need to set, default value is false

michaelhashimoto committed:
LRCI-3427 Add portal log failures to Testray results if available

ling-alan-huang committed:
POSHI-471 Fix test cases due to new Poshi Script syntax

michaelhashimoto committed:
LRCI-3427 SF

shuyangzhou committed:
LPS-175724 Cleanup bad ServiceTracker usages.

JanisZhang committed:
LPS-175217 Remove stream usages in MBMessageModelDocumentContributor

cgoncas committed:
LPS-175384 New JSONObjectIndividualSegmentsExpressionVisitorImpl class that implements a IndividualSegmentsExpressionBaseVisitor for AC Event Criteria Expressions and returns a JSON that the segments-web frontend can read.

cgoncas committed:
LPS-175384 Add tests

cgoncas committed:
LPS-175384 New SegmentsCriteriaJSONObjectMapperImpl that returns a JSON from an Segment Criteria with AC grammar.

cgoncas committed:
LPS-175384 Use the new AC SegmentsCriteriaJSONObjectMapper in the EventSegmentsCriteriaContributor.

cgoncas committed:
LPS-175384 Add tests

nNilton committed:
LPS-173287 simplify

BarbaraCabrera committed:
LPS-168842 Replace Liferay-ui:icon with clay:icon in page.jsp

victorg1991 committed:
LPS-167709 Send the portlet id in the request, otherwise the editorConfig is not created properly


The portlet id is used here to get the config contributors: https://github.com/liferay/liferay-portal/blob/c1e417111c005565e5d02bae31ce89fc44f809a7/portal-impl/src/com/liferay/portal/editor/configuration/BaseEditorProvider.java#L60-L68

victorg1991 committed:
LPS-167709 Update styles for the iframe to fill the whole height

michaelhashimoto committed:
LRCI-3427 Fix truncation issue

gamerson committed:
LPS-175300 remove default assets configuration and add node build to all

Commit:
LPS-161981 batch-engine - technical json - add actions so machins understand where to submit this batch file

Commit:
LPS-161981 batch-engine - TEST UNIT - technical json - test actions

Commit:
LPS-161981 batch-engine - no need for this - it was my leftover that I forgot to remove after finalizing test

michaelhashimoto committed:
LRCI-3427 Always try to report the error details & stacktrace

michaelhashimoto committed:
LRCI-3427 Remove unnecessary functional test failures

nNilton committed:
LPS-173287 use before and after

nNilton committed:
LPS-173287 sort

patriciajeanperez committed:
LRQA-79325 Add test suite that will run USM tests against supported environments

michaelhashimoto committed:
LRCI-3427 Create a separate Portal Log case result class

michaelhashimoto committed:
LRCI-3429 Report JUnit test failures even when tests fail in setup class

fmarcio committed:
LRAC-12959 add 'expanded' property in data type column

MirnaGama committed:
LPS-175631 Use a local service to get the file entry so the permissions won't be checked

jeffwu0724 committed:
LPS-175644 remove stream of usage in KQLQuery

jeffwu0724 committed:
LPS-175644 remove stream of usage in SharepointExtRepository

nkkwon committed:
LPS-165988 Add path for date input in filters for Object fields

nkkwon committed:
LPS-165988 Add macro for aggregation filter options

nkkwon committed:
LPS-165988 Add RangeIsAvailableForDateFields test

michaelhashimoto committed:
LRCI-3443 Ensure process is destroyed when a process is supposed to timeout

julschong committed:
LPS-175650 Remove Stream usages in DDMFormFieldTypeUtil

ces-huynguyen committed:
LPS-175501 The image version has been changed, but the image remains the same

brianchandotcom committed:
LPS-175384 SF

JanisZhang committed:
LPS-175660 Remove Stream usages in EntityFieldsMapFactory

JanisZhang committed:
LPS-175660 Remove Stream usages in EntityFieldsUtil

dantewang committed:
LPS-175217 SF

slnn committed:
LPS-172884 Remove the with(AMAttribute<AMImageProcessor, V>, Optional<V>) since it is only used by tests

slnn committed:
LPS-172884 Baseline

victorg1991 committed:
LPS-169533 Clean css of asset-publisher-web

ambrinchaudhary committed:
LPS-167436 Remove extra fieldset. It was adding the same legend than the <aui:form> does when there is no label

ambrinchaudhary committed:
LPS-167436 SF

Commit:
LPS-172534 Import Master Templates as private as well

cgoncas committed:
LPS-175384 Improve readability.

cgoncas committed:
LPS-175384 Apply the same solution to all the tests.

MirnaGama committed:
LPS-175631 Create new test method for DLAppLocalService

Stephy6 committed:
LPS-173028 Replace dashes with slashes so we can handle the timezone

eduardo-diniz committed:
LPS-175682 Fix Activities title

claraizquierdo committed:
LPS-175802 Support all event criteria

Commit:
COMMERCE-10761 - Check if the GroupFilter is not enabled to the country before add the address to dropdown options

rafaellajordao committed:
LPS-174317 add ignore to tests that needs refactor

Brunno0liveira committed:
COMMERCE-10639 AssertElementPresentNoWaitForSPAReFresh Function

Commit:
LPS-175727 Add PicklistExportImport.testcase and stubs

julschong committed:
LPS-174904 Use filter instead of transform

Commit:
LPS-162313 Add SelectHiddenPageInLinkToPageField test

Commit:
LPS-173521 Create Layout Filter

Commit:
LPS-175739 Checkbox are not being displayed on the page

Commit:
LPS-175746 Redirect URL generated by OAuth2RedirectURIInterpolator contains `:80` when request is secure and forwarded port is 80

Commit:
LPS-167846 Error when filtering a specific Environment Type

CalumR23 committed:
POSHI-481 Add check to shorten script for better parsing

julschong committed:
LPS-174904 Return all commerceChannelIds instead of empty long array when CommerceChannelAccountEntryRel is null and return curCommerceChannelAccountEntryRel.getCommerceChannelId instead of commerceChannelAccountEntryRel.getCommerceChannelId

Commit:
LPS-175739 Fix for employee should not view an organization page

jeffwu0724 committed:
LPS-175750 create patch module for spring-context 4.3.30

jeffwu0724 committed:
LPS-175750 copy DataBinder.java from spring-context

jeffwu0724 committed:
LPS-175750 apply the fix in https://github.com/spring-projects/spring-framework/commit/833e750175349ab4fd502109a8b41af77e25cdea#diff-1f22c41307a3ddcec8f1bc7a237d3b77ac7564883f4c663402993affac8a5756L25 to our patch

danielCamara02 committed:
LPS-174885 Add CanResolveConflictingPublicationsAutomatically test

kenjiheigel committed:
POSHI-473 Reword

claraizquierdo committed:
LPS-175802 Add translations

kenjiheigel committed:
POSHI-476 Rename

kenjiheigel committed:
POSHI-473 POSHI-481 POSHI-476 Use com.liferay.poshi.core:1.0.96

kenjiheigel committed:
POSHI-471 Temporarily ignore SF on these Poshi files

Commit:
LPS-175559 rest-builder - id is not valid batch parameter

Commit:
LPS-175559 commerce-headless-admin-catalog - Autogenerate - regenerate after restBuilder was fixed

Commit:
LPS-175559 commerce-headless-admin-catalog - baseline

kenjiheigel committed:
POSHI-471 SF

joelrocha68 committed:
LPS-175771 Add JSONAPI deleteProject and getProject

joelrocha68 committed:
LPS-175771 SF

xbrianlee committed:
LPS-175772 Update CSS assertion

Commit:
LPS-175760 Hide project Status and End date on partner project cards

brookedalton committed:
LPS-175774 Add service builder batch to search and publications suite

javalosjr96 committed:
LPS-175744 Change to fetch and check if null

Kvale1733 committed:
LRQA-79526 Use TextArea#Any path for blacklistModuleAndComponentConfigurationmacro

allen-nagtalon committed:
LPS-174856 Set focus back to top frame after attaching file

wanderlast committed:
LPS-173722 - refetch user to circumvent potential StaleObjectStateException

joshchong committed:
LPS-175343 Add back Weblogic 14.1.1 upgrade batch now that the upgrade failure is no longer happening on that app server

joshchong committed:
LPS-175343 Move test quarantines to test.properties

joshchong committed:
LPS-175343 Remove workaround and quarantine from WebSphere instead to not affect other environments

lorenzolapescara committed:
COMMERCE-10809 Update SystemSettings#KEY

AliciaGarciaGarcia committed:
LPS-175779 label is "Expiring Soon"

AliciaGarciaGarcia committed:
LPS-175779 buildLang

cgoncas committed:
LPS-175802 SF.

crodriguezy committed:
LPS-175799 Check if file exists

cgoncas committed:
LPS-175802 Simplify.

abhnerramos committed:
LPS-160757 Add assertJavaScriptErrors function

abhnerramos committed:
LPS-160757 Add ProxyPathRedirect test

Commit:
LPS-175736 Created logic Apply Filter Button Action

thaynam committed:
LPS-173470 Add field with total mdf request amount

pietrobocale committed:
COMMERCE-10627 Add new paths

pietrobocale committed:
COMMERCE-10627 Add new macro updateProductPublisherConfiguration

pietrobocale committed:
COMMERCE-10627 Add new test CanAddAndRemoveProductPublisherTagFilterRules

p2kmgcl committed:
LPS-175800 Remove placeholder configuration field

lorenzolapescara committed:
COMMERCE-10809 Change path used

pyoo47 committed:
LRCI-3420 Log any timeouts that occur

pyoo47 committed:
LRCI-3420 Wait up to 5 minutes for GitHub-dev sync

claraizquierdo committed:
LPS-174317 Refactor of DateTimeInput component

BarbaraCabrera committed:
LPS-174757 Replace aui:icon with clay:icon in configuration.jsp

Commit:
LPS-174836 Set status when amount is equal or higher than 15k

welitonpe committed:
LPS-173845 Added disabled propertie to input fields and fix compareruns

pyoo47 committed:
LRCI-3420 Avoid NPE

Commit:
LPS-175867 Fixing Raylife ap ci routine tests

gamerson committed:
LPS-175825 just use the filename for BSN for client extension zips

victorhcunha committed:
LPS-175737 Fix WorkflowMetricsPerformanceByStep Path

Commit:
LPS-161981 batch-engine - sort out JSON keys

joshchong committed:
LPS-174900 Remove test from environment suite since it fails on most batches

brianchandotcom committed:
POSHI-473 Rename

brianchandotcom committed:
POSHI-473 prep next

brianchandotcom committed:
LPS-175750 prep next

brianchandotcom committed:
LPS-175825 SF

brianchandotcom committed:
LPS-161981 SF

rafaprax committed:
LPS-175822 Fix typo

brianchandotcom committed:
LPS-161981 Don't we need this?

brianchandotcom committed:
LRCI-3447 to .gitignore

gamerson committed:
LPS-169846 sample-custom-element-2 oauth2 application user agent name

brianchandotcom committed:
LPS-175384 #sfme

brianchandotcom committed:
LPS-175384 auto SF

brianchandotcom committed:
LPS-175559 prep next

brianchandotcom committed:
LPS-166481 prep next

brianchandotcom committed:
LPS-175559 LPS-166481 LPS-173738 prep next

brianchandotcom committed:
LPS-175802 regen

liuxingye committed:
LPS-170293 Add macros for view search results in KB admin

liuxingye committed:
LPS-170293 Add test for search for KB article in admin search bar

brianchandotcom committed:
LPS-167709 SF

pyoo47 committed:
Revert "LRCI-3420 Wait up to 5 minutes for GitHub-dev sync"

This reverts commit 03cd8cdbf840d28ad5261c55f826f6ef1575a91f.

pyoo47 committed:
LRCI-3420 Change default timeout to 5 minutes

pyoo47 committed:
LRCI-3420 SF

brianchandotcom committed:
LPS-173287 Match original order

brianchandotcom committed:
LPS-173287 Why did you remove this? Rebase error?

brianchandotcom committed:
LPS-173287 Rename

liuxingye committed:
LPS-170293 Add disabled paths

liuxingye committed:
LPS-170293 Add macro for view active actions in management toolbar

liuxingye committed:
LPS-170293 Add test for management toolbar displays correct actions in kb folder

brianchandotcom committed:
LPS-173287 SF

brianchandotcom committed:
LPS-173287 Make it work

brianchandotcom committed:
LPS-173287 auto SF

brianchandotcom committed:
LPS-175217 Rename

brianchandotcom committed:
LRCI-3420 prep next

liuxingye committed:
LPS-170293 Add macro for open kb templates by URL

joshchong committed:
LPS-172155 Add new search-lxc suite

joshchong committed:
LPS-172155 Add test.name.skip.portal.instance to be explicit about running on default instance

liuxingye committed:
LPS-170293 Update macro for assert default message and disabled search bar

brianchandotcom committed:
LPS-175746 SF

liuxingye committed:
LPS-170293 Add macro for view active actions from template management toolbar

liuxingye committed:
LPS-170293 Add test for view correct actions from kb templates management toolbar

brianchandotcom committed:
artifact:ignore oauth2-provider-api 12.0.1 prep next

brianchandotcom committed:
U65

brianchandotcom committed:
LPS-175559 Bump for breaking change

liuxingye committed:
LPS-170293 Add macro for view actions from kb templates preview

brianchandotcom committed:
LPS-175300 prep next

brianchandotcom committed:
LRCI-3427 prep next

liuxingye committed:
LPS-170293 Update macro for edit template from toolbar

allen-ziegenfus committed:
ISSITES-5914 Undo

liuxingye committed:
LPS-170293 Add test for edit template via toolbar

Commit:
LPS-175480 Modify macro to make it reusable

dantewang committed:
LPS-175213 SF

liuxingye committed:
LPS-170293 Add path for view kb article in admin more specific

liuxingye committed:
LPS-170293 Update macro for view kb article in admin more specific

dantewang committed:
LPS-175212 SF

liuxingye committed:
LPS-170293 Add test for sort articles by alphabetical order

brianchandotcom committed:
LPS-175559 Regen a lot more?

brianchandotcom committed:
LPS-175644 SF

liferay-translation committed:
LPS-77699 Update Translations

brianchandotcom committed:
LPS-168540 Regen

brianchandotcom committed:
LPS-174317 Wordsmith

brianchandotcom committed:
LPS-174317 Regen

brianchandotcom committed:
COMMERCE-10639 SF

Commit:
COMMERCE-10761 - CommerceChannelRel and CommerceChannelRelLocalServiceUtil added to init JSP file

Commit:
COMMERCE-10761 - Add a check to see if the country is filtered for the channel before put the address into dropdown

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.
